linkernetworks/azure-intelligent-edge-patterns
# Backend/peter dev

## Purpose
* Modify Error message when duplicate part name create/update
* Demo PartIdentification can now use REAL Camera

## Does this introduce a breaking change?
```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
```

* Test the code
  - Create one part
  - Create Another part with same name => Check error log
  - Create another part with different name. Save. Change name to duplicate and save again => Check error log

## Other Information
